### PR TITLE
virtualenv invocation removal

### PR DESCRIPTION
**Pull request by lsh-0**
replaces 'virtualenv' invocations with 'python3 -m venv venv'

### CODE REVIEW

**Comment by giorgiosironi** [comment]
The `pip install` process seems to have some problem? Are we missing some `python3-*` OS package on `elife-libraries` nodes?

**Comment by lsh-0** [comment]
not sure, I'll investigate today

**Comment by lsh-0** [comment]
I couldn't figure out what was going on, something to do with pylint+astroid I think. I ended up just wiping out the version constraints and letting pip decide what to install and then froze it at that.

File: requirements.txt [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-py==1.4.31
-pylint==1.7.4
-pytest==3.0.3
-six==1.10.0
+mock==3.0.5
+pbr==5.4.4
+py==1.8.1
+pylint==2.4.4
+pytest==5.3.2
+six==1.13.0
```
Comment (giorgiosironi) [inline requirements.txt]: still needed?

Comment (lsh-0) [reply requirements.txt]: six? good point. I'll remove it and see what happens

Comment (lsh-0) [reply requirements.txt]: done

**Comment by lsh-0** [comment]
waiting on a review from @gnott

**Pull request merged**